ui/components-react/src/components-react/converters/TypeConverterManager.ts

```diff
@@ -50,10 +55,11 @@ export class TypeConverterManager {
     if (TypeConverterManager._converters.hasOwnProperty(fullConverterName))
       return TypeConverterManager._converters[fullConverterName];
 
-    if (!TypeConverterManager._defaultTypeConverter) {
-      const { StringTypeConverter } = require("../converters/StringTypeConverter"); // eslint-disable-line @typescript-eslint/no-var-requires
-      TypeConverterManager._defaultTypeConverter = new StringTypeConverter();
-    }
     return TypeConverterManager._defaultTypeConverter;
   }
 }
+
+// register string type converters here to avoid circular dependency in the StringTypeConverter module
```

Comment:
Since we removed the "new StringTypeConverter" line completely (previous line 55), there is no need to move the imports here as the "StringTypeConverter" type is no longer needed in that file, which also removes the circular depenency.

Reply:
meant to move those back after this change. Will do now.

Reply:
done